                                                                                                                                                              ERROR:daisy.client:Block downscale/53258 with read ROI [9600:9700, 9728:9984, 6656:6912] (100, 256, 256) and write ROI [9600:9700, 9728:9984, 6656:6912] (100, 256, 256) failed in worker 1405
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
downscale ▶:   0%|                                                                                  | 0/7056 [00:40<?, ?blocks/s, ⧗=0, ▶=48, ✔=0, ✗=6789, ∅=0]                                                                                                                                                              ERROR:daisy.client:Block downscale/15139 with read ROI [9600:9700, 10240:10496, 0:256] (100, 256, 256) and write ROI [9600:9700, 10240:10496, 0:256] (100, 256, 256) failed in worker 1405
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
downscale ▶:   0%|                                                                                  | 0/7056 [00:40<?, ?blocks/s, ⧗=0, ▶=48, ✔=0, ✗=6789, ∅=0]                                                                                                                                                              ERROR:daisy.client:Block downscale/35213 with read ROI [9600:9700, 10496:10588, 3584:3840] (100, 92, 256) and write ROI [9600:9700, 10496:10588, 3584:3840] (100, 92, 256) failed in worker 1405
Traceback (most recent call last):
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/client.py", line 85, in acquire_block
    yield block
  File "/Users/joelyancey/.local/share/virtualenvs/swift-ir-AuCIf4YN/lib/python3.9/site-packages/daisy/task.py", line 179, in _process_blocks
    self.process_function(block)
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 688, in <lambda>
    process_function=lambda b: downscale_block(
  File "/Users/joelyancey/glanceem_swift/swift-ir/alignEM/package/scale_pyramid.py", line 640, in downscale_block
    out_data = skimage.measure.block_reduce(in_data, factor, np.mean)
NameError: name 'skimage' is not defined
downscale ▶:   0%|                                                                                  | 0/7056 [00:40<?, ?blocks/s, ⧗=0, ▶=48, ✔=0, ✗=6789, ∅=0]